### DISCUSSION TITLE
Can NOT request and transfer files between LCRC and NERSC-HPSS

### DISCUSSION THREAD

**Discussion by keziming** [E3SM-Project/zstash#320] [Questions]
### Question criteria

- [X] I added a descriptive title here.
- [X] I searched the [zstash GitHub Discussions](https://github.com/E3SM-Project/zstash/discussions) to find a similar question and didn't find it.
- [X] I searched the [zstash documentation](https://e3sm-project.github.io/zstash).

### What is the deadline?

no deadline

### Describe your question

For example, If I want to get the restart files from Nersc-HPSS, I use following command
zstash extract --hpss=globus://9cd89cfd-6d04-11e5-ba46-22000b92c6ec/home/f/forsyth/E3SMv2/v2.LR.amip_0101/archive/rest

But the error is 
globus_sdk.services.transfer.errors.TransferAPIError: ('POST', 'https://transfer.api.globus.org/v0.10/endpoint/61f9954c-a4fa-11ea-8f07-0a21f750d19b/autoactivate?if_expires_in=600', 'Bearer', 404, 'EndpointNotFound', "No such endpoint '61f9954c-a4fa-11ea-8f07-0a21f750d19b'", 'QIvqfkd2U')

The same command is working months ago. I have no issue for log into globus website and log into the NERSC and LCRC through endpoints. Could anyone kindly help me?

Thank you very much!

Ziming
LLNL AEED

### What machine were you running on?

LCRC

### Environment

bash

**Comment by chengzhuzhang** [comment]
Hi @keziming, I'm wondering if this is caused by the same problem that the current default LCRC Globus endpoint got expired on Dec 1, and is replaced by `improv`: https://github.com/E3SM-Project/zstash/discussions/317, https://github.com/E3SM-Project/zstash/issues/318
Could you try the solution Ryan @forsyth2 posted here: https://github.com/E3SM-Project/zstash/discussions/317#discussioncomment-7720625? Ryan also had a fix in zstash to handle expired endpoint and was merged to main branch.

**Reply by forsyth2** [reply]
Yes, this is almost certainly the cause. Thanks!

**Comment by keziming** [comment]
@chengzhuzhang Thank you very much! I will try it and let you know.
